test/parser/testlogfileparser.py

```diff
@@ -25,48 +26,68 @@
 
 class FileWrapperTest(unittest.TestCase):
 
+    def check_seek(self, wrapper):
+        """Check that a FileWrapper can seek properly"""
+        wrapper.seek(0, 2)
+        assert wrapper.pos == wrapper.size        
+        
+        wrapper.seek(0, 0)
+        assert wrapper.pos == 0
+        
+        with pytest.raises(NotImplementedError):
+            wrapper.seek(0, 1)
+
     def test_file_seek(self):
         """Can we seek anywhere in a file object?"""
         fpath = os.path.join(__datadir__,"data/ADF/basicADF2007.01/dvb_gopt.adfout")
         with open(fpath, 'r') as fobject:
             wrapper = cclib.parser.logfileparser.FileWrapper(fobject)
-            wrapper.seek(0, 0)
-            assert wrapper.pos == 0
-            wrapper.seek(10, 0)
-            assert wrapper.pos == 10
-            wrapper.seek(0, 2)
-            assert wrapper.pos == wrapper.size
+            self.check_seek(wrapper)
+#             wrapper.seek(0, 0)
```

Comment:
Let's just get rid of all the commented blocks.  That will make it clearer how much of an improvement this is.